packages/gatsby-source-strapi/package.json

```diff
@@ -0,0 +1,70 @@
+{
+  "name": "gatsby-source-strapi",
+  "version": "3.0.0",
+  "description": "Gatsby source plugin for building websites using Strapi as a data source",
+  "author": {
+    "email": "hi@strapi.io",
+    "name": "Strapi Solutions",
+    "url": "http://strapi.io"
+  },
+  "maintainers": [
+    {
+      "name": "Strapi Solutions",
+      "email": "hi@strapi.io",
+      "url": "http://strapi.io"
+    }
+  ],
+  "directories": {
+    "lib": "./lib"
+  },
+  "files": [
+    "lib/*",
+    "gatsby-node.js",
+    "index.js"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/gatsby-uc/gatsby-source-strapi.git"
+  },
+  "bugs": {
+    "url": "https://github.com/gatsby-uc/gatsby-source-strapi/issues"
+  },
+  "license": "MIT",
+  "scripts": {
+    "prepublishOnly": "npm run build",
+    "build": "babel src --out-dir lib",
+    "watch": "babel -w src --out-dir lib",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "format": "prettier --write 'src/**/*.js'",
+    "lint": "eslint src"
+  },
+  "keywords": [
+    "gatsby",
+    "gatsby-plugin",
+    "gatsby-source-plugin",
+    "strapi"
+  ],
+  "dependencies": {
+    "axios": "0.24.0",
+    "commonmark": "^0.30.0",
+    "gatsby-source-filesystem": "4.5.1",
+    "lodash": "^4.17.21",
+    "qs": "^6.10.3"
+  },
+  "devDependencies": {
+    "@babel/cli": "7.14.5",
+    "@babel/core": "7.14.6",
+    "@babel/plugin-transform-runtime": "7.14.5",
+    "@babel/preset-env": "7.14.7",
+    "eslint": "7.13.0",
+    "husky": "4.3.0",
+    "lint-staged": "10.5.1",
+    "prettier": "2.1.2"
```

Comment:
I believe these can all be removed. Either the repo provides them or in Babel's case moving to the Gatsby preset means we wont use these directly.